tsconfig.json

```diff
@@ -8,7 +9,7 @@
     ],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
Is this change intentional?

Reply:
I was checking and this change is not needed, so I reverted